CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 1.2.1 (2016-04-22)
+
+#### API
+
+Use engine-api to handle large number of API calls
+Update ContainerConfig to embed HostConfig and NetworkingConfig
+stop/restart/kill an non-existent container should returns 500 rather than 404
+Return an error when assertion fails in hijack
+Return error when Image Pull fails
+Fix image pull bug (wait until download finishes)
+
+#### Build
+
+Switch to go 1.6.2
```

Comment:
Didn't realize Swarm was on Go 1.6.x already. Will this be a problem for older clients connecting to the Swarm API over a socket connection? See https://github.com/docker/docker/issues/20865